js/libs/form.js

```diff
@@ -339,7 +339,9 @@ Fliplet().then(function() {
             } else if (field._type === 'flDateRange' && ['default', 'always'].indexOf(fieldSettings.autofill) > -1) {
               value = fieldSettings.defaultSource === 'submission' ? moment().locale('en').format('YYYY-MM-DD') : $vm.today;
```

Comment:
Is this not a problem as well?

```suggestion
              value = fieldSettings.defaultSource === 'submission' ? moment().locale('en').format('YYYY-MM-DD') : { start: $vm.today, end: $vm.today };
```